docs/docs/features/backlight.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Backlight
+sidebar_label: Backlight
+---
+
+Backlight is a feature used to control array of LEDs, usually placed through or under switches. Unlike [RGB Underglow](./underglow), backlight currently allows only one color per LED, also LEDs are not addressable, so you can't control individual LEDs.
+
+## Enabling Backlight
+
+To enable backlight on your board or shield, simply enable the `CONFIG_ZMK_BACKLIGHT` configuration values in the `.conf` file of your user config directory as such:
+
+```
+CONFIG_ZMK_BACKLIGHT=y
+```
+
+If your board or shield does not have backlight configured, refer to [Adding Backlight to a Board](#adding-backlight-to-a-board).
+
+## Configuring Backlight
+
+There are various Kconfig options used to configure the backlight feature. These can all be set in the `.conf` file.
+
+| Option                      | Description                                             | Default |
+| --------------------------- | ------------------------------------------------------- | ------- |
+| `ZMK_BACKLIGHT_LEVELS`      | Number of brightness levels                             | 6       |
+| `ZMK_BACKLIGHT_LEVEL_START` | Default brightness level (1 turn off backlight)         | 3       |
+| `ZMK_BACKLIGHT_ON_START`    | Default backlight state                                 | y       |
+| `ZMK_BACKLIGHT_SLEEP`       | Turn off backlight when keyboard goes into sleep status | y       |
+| `ZMK_BACKLIGHT_USB`         | Turn off backlight when USB is disconnected             | n       |
+
+## Adding Backlight to a Board
+
+First, you need to enable PWM by adding the following lines to your `.overlay` file:
```

Comment:
A similar description as to why this needs to go into an overlay/board configuration like the RGB underglow one might be useful.